src/mcode.ts

```diff
@@ -1110,17 +1141,19 @@ export class ExtractedMCODE {
       return 'NOT_SURE';
     }
   }
-  getMedicationStatementValue(): string {
+  getMedicationStatementValues(): string[] {
+    var medicationValues:string[] = [];
```

Comment:
Same as above: this should likely be `const`.

Reply:
Changed.